lib/commonConfig.js

```diff
@@ -91,7 +91,8 @@ function processEnvironmentVariables() {
              'IOTA_APPEND_MODE',
```

Comment:
CHANGES_NEXT_RELEASE entry should be provided for this PR.

Reply:
Fixed b41a18a